css/styles.css

```diff
@@ -111,11 +111,11 @@ i {
   padding-right: 10px;
   padding-bottom: 10px;
   cursor: pointer;
+  transition: transform 0.25s ease;
```

Comment:
You're not doing any `transform` effects, so this `transition` isn't doing anything.